src/app/notes/notes-view/notes-view.component.ts

```diff
@@ -1,15 +1,21 @@
 import { Component, OnInit } from '@angular/core';
+import { NotesService } from '../notes.service';
+import { Note } from '../note.model';
 
 @Component({
   selector: 'app-notes-view',
   templateUrl: './notes-view.component.html',
-  styleUrls: ['./notes-view.component.scss']
+  styleUrls: ['./notes-view.component.scss'],
 })
 export class NotesViewComponent implements OnInit {
+  selectedNote: Note;
 
-  constructor() { }
+  constructor(private notesService: NotesService) { }
 
   ngOnInit() {
+    this.notesService.noteSelected.subscribe((note: Note) => {
```

Comment:
tutaj jest problem z ta asynchronicznoscia.

Do zrobienia w notesService
pole initialStoreValue = > zamockowana wartosc
prywante pole notesStore = new BehaviorSubject(initialStoreValue)
public observable  np obvervable$ ($konwencja nazewnictwa dla observable'a) obvervable$ = this.notesStore.asObservable();